base/applications/cmdutils/cscript/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ add_typelib(ihost.idl)
 set_source_files_properties(rsrc.rc PROPERTIES OBJECT_DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/ihost.tlb)
 target_link_libraries(cscript uuid wine)
 set_module_type(cscript win32cui UNICODE)
+add_delay_importlibs(cscript shlwapi)
```

Comment:
Delayed import of shlwapi might be fine (I can see that, at least transitively, being delay-loaded also on XPSP3).
Still the question for documentation purposes: Which exact call introduces the need for shlwapi.dll in your code?

The latest changeset brings me back to neutrality regarding this PR, thank you!

Reply:
> Which exact call introduces the need for shlwapi.dll in your code?

UrlCreateFromPathW